scripts/benchmarks_ci.py

```diff
@@ -225,6 +227,38 @@ def __is_valid_datetime(dt: str) -> str:
         action='store_true',
         help='Turns off verbosity.',
     )
+    parser.add_argument(
+        '--build-only',
+        dest='build_only',
+        required=False,
+        default=False,
+        action='store_true',
+        help='Builds the benchmarks but does not run them.',
+    )
+    parser.add_argument(
+        '--run-only',
+        dest='run_only',
+        required=False,
+        default=False,
+        action='store_true',
+        help='Attempts to run the benchmarks without building.',
+    )
+    parser.add_argument(
```

Comment:
I believe this is not needed because it was already added to the `micro_benchmarks` module. It will be already imported, and this will overwrite it.